sdk/python/setup.py

```diff
@@ -29,6 +29,7 @@
     'cryptography>=2.4.2',
     'google-auth>=1.6.1',
     'requests_toolbelt>=0.8.0',
+    'kfp-server-api==0.1.18.2',
```

Comment:
Fix this to >= to avoid conflicts down the line.

Reply:
Our API is usually backwards compatible, so using older client is fine.
I'd say >= can be a bigger source of conflicts.
I'm afraid that a new generated code can break older SDKs people might have. I see no harm in tying the packages 1:1 like they were tied before.

Reply:
For the time being, kfp is the only package depending on kfp-server-api.
kfp-server-api is auto-generated and we currently do not have tests to ensure it's backwards compatible to some SDK version we've released 2 months ago.
Without upper bound, we can never introduce a breaking change.
I think it's better to have 1:1 relation for now until we figure this out.

Reply:
Talked offline.

The main requirement from me was that `pip3 install kfp=0.1.18` never breaks (with us releaseng new versions of `kfp-server-api`). It's only a matter of time before a breaking change is introduced and I do not want the installation of older SDK packages to break.

We've agreed to have the upper version limit that we'll extend whenever a new api package gets released. Version span also allows bugfixes for the api package to be released.
`kfp-server-api >= 0.1.18, < 0.1.19`